sql/_05_plcsql/_01_testspec/_01_basic_structure/_02_create_function/_12_comment/cases/01_comment.sql

```diff
@@ -0,0 +1,14 @@
+--+ server-message on
+
+create function comment_test() return string as
+begin
+    return '';
+end
+comment 'comment test';
+
+--select comment_test() as "col" from dual;
+select * from _db_stored_procedure where sp_name='comment_test';
+
```

Comment:
select sp_name, comment from _db_stored_procedure where sp_name='comment_test';

Reply:
I think, need to check sp_type, return_type, arg_count, and target too.
if that column is checked by another tc then I will reflect that.

Reply:
we are decision the use a '*'.